swingset/src/main/java/com/nqadmin/swingset/SSDataNavigator.java

```diff
@@ -848,6 +894,27 @@ protected void addToolTips() {
 		deleteButton.setToolTipText("Delete Record");
 
 	} // end protected void addToolTips() {
+	
+	/**
+	 * Common code to commit changes to the database from the rowset if
+	 * modifications are allowed.
+	 * @return true unless dBNav.allowUpdate() returns false
+	 * @throws SQLException SQL Exception if rowset call to updateRow() fails
+	 */
+	private boolean commitChanges() throws SQLException {
```

Comment:
Cool. It's nice to have that dup'd code gone. I guess this is the guts of the "auto commit"

(I usually just hit add acomment, on the last comment I pushed "start review" to try something new, and got a yellowish button that said pending. I guess that makes sense. Get rid of that soon.